.github/workflows/basic-qa.yml

```diff
@@ -127,8 +127,7 @@ jobs:
           custom-cache-suffix: $(date -u "+%Y-%m")
 
       - name: "Set PHPCS version (lowest)"
-        if: ${{ matrix.phpcs_version == 'lowest' }}
```

Comment:
Shoot! This condition shouldn't have been removed... Must have accidentally happened during on of the rebases....

I will fix this up.